.github/workflows/build.yml

```diff
@@ -1,128 +0,0 @@
-# GitHub Actions Workflow created for testing and preparing the plugin release in following steps:
-# - validate Gradle Wrapper,
-# - run test and verifyPlugin tasks,
-# - run buildPlugin task and prepare artifact for the further tests,
-# - run IntelliJ Plugin Verifier,
-#
-# Workflow is triggered on push and pull_request events.
-#
-# Docs:
-# - GitHub Actions: https://help.github.com/en/actions
-# - IntelliJ Plugin Verifier GitHub Action: https://github.com/ChrisCarini/intellij-platform-plugin-verifier-action
-
-name: Build
-on: [push, pull_request]
-jobs:
-
-  # Run Gradle Wrapper Validation Action to verify the wrapper's checksum
-  gradleValidation:
-    name: Gradle Wrapper Validation
-    runs-on: ubuntu-latest
-    steps:
-
-      - name: Fetch Sources
-        uses: actions/checkout@v2
-
-      - name: Gradle Wrapper Validation
-        uses: gradle/wrapper-validation-action@v1
```

Comment:
I completely got rid of this job in the new, unified workflow. I don't see a lot of value in this validation and it takes forever to finish like everything gradle & java...